lib/Backend/Inline.cpp

```diff
@@ -5334,11 +5334,19 @@ Inline::Simd128FixLoadStoreInstr(Js::BuiltinFunction builtInId, IR::Instr * call
         IR::Opnd *linkOpnd = callInstr->GetSrc1();
         IR::Instr *eaInstr1, *eaInstr2, *eaInstr3;
         IR::Opnd *value, *index, *arr;
+        IR::Opnd *dst = callInstr->GetDst();
 
         eaInstr1 = linkOpnd->GetStackSym()->m_instrDef;
         value = eaInstr1->GetSrc1();
         linkOpnd = eaInstr1->GetSrc2();
 
+        //Load value to be stored to dst. Store returns the value being stored.
+        if (dst)
+        {
+            IR::Instr * ldInstr = IR::Instr::New(Js::OpCode::Ld_A, dst, value, callInstr->m_func);
+            callInstr->InsertAfter(ldInstr);
```

Comment:
Nit: Can this be changed to InsertBefore and done at the end? InsertAfter is hard to read.